docs/reference/mapping.asciidoc

```diff
@@ -5,30 +5,54 @@
 --
 
 Mapping is the process of defining how a document, and the fields it contains,
-are stored and indexed. For instance, use mappings to define:
+are stored and indexed.
 
-* which string fields should be treated as full text fields.
-* which fields contain numbers, dates, or geolocations.
-* the <<mapping-date-format,format>> of date values.
-* custom rules to control the mapping for
-  <<dynamic-mapping,dynamically added fields>>.
+Each document is a collection of fields, which each have their own
+<<mapping-types,data type>>. When mapping your data, you create a mapping
+definition, which contains a list of fields or `properties` that are pertinent
+to the document. A mapping definition also includes
+<<mapping-fields,metadata fields>>, like the `_source` field, which customize
+how a document's associated metadata is handled.
+
+You can choose a _dynamic mapping_ or an _explicit mapping_ to define your data.
```

Comment:
one thing that I am not sure is clear is that you can also go for a mixed approach, meaning you map some fields manually, and others dynamically. From this sentence it may seem like you need to decide to go for one or the other approach upfront.

Reply:
Yeah, that sentence is limiting. Is this revision precise enough?

>Use _dynamic mapping_ and _explicit mapping_ to define your data. Each method
provides different benefits based on where you are in your data journey. You
might explicitly map specific fields that you want to query frequently, such as
a `timestamp`, whereas {es} can add other fields dynamically.

Reply:
well, even the timestamp does not need to be mapped if its format is the one that ES supports by default. It is really "map anything explicitly if the defaults are not good enough, or if you want to have control over which fields get created"